Global/Eclipse.gitignore

```diff
@@ -49,3 +49,15 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Java class files
+**/*.class
+**/classes
+**/classes/**
+
+# svn plugin
+**/.svn
+**/.svn/**
+
+# virtual machine crash logs
```

Comment:
Sorry,I've made a spelling mistake.It should be java virtual machine crash logs.




在 2016-05-01 16:56:20，"Brendan Forster" <notifications@github.com> 写道：


In Global/Eclipse.gitignore:

> @@ -49,3 +49,15 @@ local.properties
>  
>  # Code Recommenders
>  .recommenders/
> +
> +# Java class files
> +**/*.class
> +**/classes
> +**/classes/**
> +
> +# svn plugin
> +**/.svn
> +**/.svn/**
> +
> +# virtual machine crash logs


Which sort of virtual machine is this?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub